app/views/refinery/authentication/devise/admin/users/_actions.html.erb

```diff
@@ -1,7 +1,5 @@
-<ul>
-  <li><%= render '/refinery/admin/search', :url => refinery.authentication_devise_admin_users_path -%></li>
```

Comment:
Yes, next PR should add search in from refinerycms-search.  That's what was discussed, right?